### ISSUE TITLE
Add function on client to open/close the pause menu map in fullscreen

### ISSUE THREAD

**Issue by EtzBetz** [altmp/altv-issues#792]
**Suggestion**

I would like a function on the client to open or close the pause menu in fullscreen directly, without any steps in between.

**Use cases**

Currently, when the player wants to look at the game's map or the game is supposed to show the map, there is no really convenient way of doing this. The player has to open the pause menu and click on the map in order for it to open in fullscreen and be interactable. If I as a server developer want to show the map, there is not really a way, I can only open the pause menu with code, but not make the map be shown in fullscreen. I'd have to implement my own webview map.

**Describe the solution you'd like**

`alt.toggleMenuMap(boolean)`

**Describe alternatives you've considered**

I'd have to implement my own webview map or cancel player input and open the menu with code and click myself with code on the map.

**Additional context**

This isn't important, it would just be a nice to have feature and it's also something that other mods don't have, which would make alt:V stand apart.

**Comment by 7Hazard** [comment]
This is just how the GTAV map works

**Comment by EtzBetz** [comment]
Yes, I'm aware of that. Do you think that it isn't feasible implementing this?
